AudibleApi.Common/_DtoBase.cs

```diff
@@ -1,11 +1,17 @@
-﻿using Newtonsoft.Json.Linq;
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using System;
 
 namespace AudibleApi.Common
 {
-	public abstract class DtoBase<T>
+	public abstract class DtoBase
 	{
-		public static T FromJson(string json) => Converter.FromJson<T>(json);
+		public JObject SourceJson { get; set; }
+	}
+
+	public abstract class DtoBase<T> : DtoBase where T : DtoBase<T>
```

Comment:
for this and _V10Base, is this `where` correct? I can't tell what it's accomplishing.

Reply:
Yup. All DTO classes are in the form of `V10Base<T>` or  `DtoBase<T>`

e.g. `Item : DtoBase<Item>`

Reply:
I get that. This `where` seems to be saying, you can only inherit from me if you inherit from me. I can't tell what it's adding.

Reply:
You're night. Totally superfluous. IDK what I was thinking.

Reply:
lol no worries. I felt like I was missing something clever but I couldn't make out what.